Jobs/GenerateTag.yml

```diff
@@ -0,0 +1,69 @@
+## @file
+# Template file used to generate tags on ADO. This template requires that the
+# consumer specifies this repository as a resource named mu_devops.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+  - name: base_version
+    displayName: The base version.
+    type: string
+    default: ""
+  - name: git_name
+    displayName: Name to use for creating tag.
+    type: string
+    default: ""
+  - name: git_email
+    displayName: Email to use for creating tag.
+    type: string
+    default: ""
+  - name: notes_file
+    displayName: Path to the notes file to generate.
+    type: string
+    default: "ReleaseNotes.md"
+
+jobs:
+  - job: Create_Release_Tag
+    steps:
+      - checkout: self
+        clean: true
+        fetchTags: true
+        persistCredentials: true
+        path: "target"
+
+      - checkout: mu_devops
+        path: "mu_devops"
+        fetchDepth: 0
+
+      - template: ../Steps/SetupPythonPreReqs.yml
+        parameters:
+          install_pip_modules: false
+

```

Comment:
maybe we should add one step here to validate the base_version, git_name and git_email parameters since consumer  should update these parameters instead of leave it as default empty string.